visual-recognition/README.md

```diff
@@ -17,21 +17,90 @@
 ```
 
 ## Usage
-Use the [Visual Recognition][visual_recognition] service to recognize the
-following picture.
+Use the [Visual Recognition][visual_recognition] service to analyze image data. Use either v3 or v4 to analyze the following image:
 
-![Car](https://visual-recognition-demo.ng.bluemix.net/images/samples/5.jpg)
+![Dog](https://visual-recognition-demo.ng.bluemix.net/images/samples/5.jpg)
+
+### Using Visual Recognition v3
 
 ```java
+// make sure to use the Visual Recognition v3 import!
+import com.ibm.watson.visual_recognition.v3.VisualRecognition;
+
 Authenticator authenticator = new IamAuthenticator("<iam_api_key>");
 VisualRecognition service = new VisualRecognition("2016-05-20", authenticator);
 
 System.out.println("Classify an image");
 ClassifyOptions options = new ClassifyOptions.Builder()
-  .imagesFile(new File(SINGLE_IMAGE_FILE))
+  .imagesFile(new File(IMAGE_FILE)) // replace with path to file
   .build();
 ClassifiedImages result = service.classify(options).execute().getResult();
 System.out.println(result);
 ```
 
+### Using Visual Recognition v4
+
+```java
+// make sure to use the Visual Recognition v4 import!
+import com.ibm.watson.visual_recognition.v4.VisualRecognition;
+
+Authenticator authenticator = new IamAuthenticator("<iam_api_key>");
+VisualRecognition service = new VisualRecognition("2019-02-11", authenticator);
+
+// create a new collection
+CreateCollectionOptions createCollectionOptions = new CreateCollectionOptions.Builder()
+    .name("example-collection")
+    .build();
+Collection exampleCollection = service.createCollection(createCollectionOptions).execute().getResult();
+String exampleCollectionId = exampleCollection.getCollectionId();
+
+// add the above image to your collection
+String imageUrl = "https://visual-recognition-demo.ng.bluemix.net/images/samples/5.jpg";
+AddImagesOptions addImagesOptions = new AddImagesOptions.Builder()
+    .addImageUrl(imageUrl)
+    .collectionId(exampleCollectionId)
```

Comment:
should imageUrl and exampleCollectionId be arrays?

Reply:
There's a method to add a single one, but the names are still wrong since I forgot to update this with the newest regen. Thanks for the catch!

Reply:
Actually this one's fine, the `AnalyzeOptions` one does need changed though.